README.md

```diff
@@ -1,51 +1,70 @@
+
+Between the use of CREATE2 opcode and the one-time-use account for the deployer, we can ensure that a given contract will exist at the exact same address on every chain, but without using the exact gas pricing or limits every time.
+
+## Encoding the deployment transaction
+
+The data should be the 32 byte 'salt' followed by the init code.
+
 ## How to get the singleton deployed to your network
 
-As the singleton is deployed with an EIP155 transaction, we need to sign the deployment transaction for your network. But some prerequisites must be met before that, and the most important one is having funds on the deployer so we can deploy the contract.
+As the singleton is deployed with an EIP155 transaction, we must sign the deployment transaction for your network. But some prerequisites must be met before that, and the most important one is having funds on the deployer so we can deploy the contract.
 
-- Make sure your network is on https://chainlist.org/ . We will not accept networks not present there.
-- Install dependencies running `yarn`.
-- Estimate the contract deployment: `RPC='http://my-rpc-address' yarn estimate`.
-- Send `requiredFunds` to the deployer address `0xE1CB04A0fA36DdD16a06ea828007E35e1a3cBC37`.
-- Open a PR and we will deploy the singleton contract.
+1. Make sure your network is on https://chainlist.org/ . We will not accept networks not present there.
+2. Create an issue following the request for a new network template.
+3. Once the issue is created, the issue will be automatically validated, and a bot will post a comment with the address of the deployer and the amount of funds needed to deploy the contract.
+4. After you have sent the funds to the deployer, mark the checkbox on the issue, and the Safe team will sign the transaction and deploy the contract.
+
+The Safe team will aim to respond to new network requests within two weeks.
 
 ## Expected Addresses
 
-For all networks the same deployer key is used. The address for this key is `0xE1CB04A0fA36DdD16a06ea828007E35e1a3cBC37`.
+For all networks, the same deployer key is used. The address for this key is `0xE1CB04A0fA36DdD16a06ea828007E35e1a3cBC37`.
+
+This results in the address for the factory being `0x914d7Fec6aaC8cd542e72Bca78B30650d45643d7` for all bytecode-compatible EVM networks.
+
+For zkSync-based networks, the same deployer is used, and the expected factory address is `0xaECDbB0a3B1C6D1Fe1755866e330D82eC81fD4FD`.
+
+**Note**: For zkSync, the factory is deployed using the `create2` method of the system deployer using the zero hash (`0x0000000000000000000000000000000000000000000000000000000000000000`).
```

Comment:
Grammar nit: I would keep these in the same paragraph since they are very related.

Reply:
I didn't change it (the diff is there because I add some lines above) I will apply the proposed change.